pkg/specgen/generate/config_linux.go

```diff
@@ -225,6 +226,11 @@ func addDevice(g *generate.Generator, device string) error {
 	if err != nil {
 		return err
 	}
+	if src == "/dev/fuse" {
+		// if the user is asking for fuse inside the container
+		// make sure the module is loaded.
+		exec.Command("modprobe", "fuse").Run()
```

Comment:
Should we make this conditional on being root? I don't think rootless will have the ability to modprobe

Reply:
Ah, is it that this function only happens as root, because rootless bind-mounts devices?

Reply:
And does this change the host or the container?  I thought the problem was the host side and I'd be hesitant to change the kernel on the host without some kind of confirmation from the user.

Reply:
rootless containers need it too, but I agree it is pointless to try it since they miss permissions for doing it.  Moving to run only as root.